android/runtime/common/src/js/module.js

```diff
@@ -617,6 +617,9 @@ Module.prototype.filenameExists = function (filename) {
 	if (!fileIndex) {
 		var json = assets.readAsset('index.json');
 		fileIndex = JSON.parse(json);
+		if (!fileIndex) {
```

Comment:
I think we want to explicitly return a Boolean in this method. I understand that `null` is "truthy" and evaluates to `false`, but I'd strongly prefer we simply alter the return statement below to be:
```javascript
return filename && (filename in fileIndex);
```